recipes/spyder/meta.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: spyder
+  version: 2.3.9
```

Comment:
With versioning, we tend to use jinja templates to reduce redundancy and lower maintenance burden. Here is an [example]( https://github.com/conda-forge/envisage-feedstock/blob/32c9b5a9b4bc2d2f37add6d52bc93cf7b7759f0d/recipe/meta.yaml ).